resource-managers/mesos/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackendSuite.scala

```diff
@@ -107,6 +107,28 @@ class MesosCoarseGrainedSchedulerBackendSuite extends SparkFunSuite
     verifyTaskLaunched(driver, "o2")
   }
 
+  test("mesos declines offers from blacklisted slave") {
```

Comment:
Since this was a bug, please use `SPARK-19755` prefix.
```scala
- test("mesos declines offers from blacklisted slave") {
+ test("SPARK-19755 mesos declines offers from blacklisted slave") {
```

Reply:
Changed